src/workers/jobs.js

```diff
@@ -759,6 +759,7 @@ export async function exportCampaign(job) {
         "contact[optOut]": optOuts.find(ele => ele.cell === contact.cell)
           ? "true"
           : "false",
+        "contact[optOutRecord]": contact.is_opted_out,
```

Comment:
Wondering if "contact[optOutRecord]" might not clear up the confusion at it's not much different than "contact[optOut]".  Maybe rename to contact[optOutCampaign] or similar to differentiate as the campaign contact opt out flag.